### DISCUSSION TITLE
Does not support select using cursor in file finding window?

### DISCUSSION THREAD

**Discussion by juniway** [helix-editor/helix#2185] [Ideas] [answered]
when pressing `space` + `f`, we enter into the file finding window, but I can't use the cursor to select (to copy) in this window.

<img width="1191" alt="CleanShot 2022-04-20 at 13 23 13@2x" src="https://user-images.githubusercontent.com/3980841/164156613-f72ba6d2-8e83-4b82-ba93-c445b4fd246f.png">

**Comment by pickfire** [comment]
It is just a file picker, you have to go into the file, do the selection, then `gm` (goto last modified file) or `ga` (goto last accessed file) and do whatever you want, it just finds the file for you.

**Comment by juniway** [comment]
sorry, I mean using mouse to select.

**Comment by pickfire** [comment, marked as answer]
Converted this to an issue, it can be an issue instead of discussion, not sure who is interested to do it but feel free to work on it.

https://github.com/helix-editor/helix/issues/2235
